src/view/com/composer/text-input/mobile/Autocomplete.tsx

```diff
@@ -19,7 +21,13 @@ export const Autocomplete = observer(function AutocompleteImpl({
   const positionInterp = useAnimatedValue(0)
   const {getGraphemeString} = useGrapheme()
   const isActive = !!prefix
-  const {data: suggestions} = useActorAutocompleteQuery(prefix)
+  const {data: suggestions, isFetching} = useActorAutocompleteQuery(prefix)
+  const suggestionsRef = useRef<
```

Comment:
Why is this needed?

Reply:
To rephase, why is the code below not using `suggestions` directly?

Reply:
Looks like it's so it renders the old results while new results are being fetched. I'll add a comment

Reply:
Looks like it's working as intended without though 🤔

Reply:
Yep, to show old results while new ones are being fetched

Reply:
I see. The canonical pattern for this looks a bit different (writing to refs during render isn't supported by React) but by the time React adds a warning/error for this we'll probably rewrite this five times anyway so lg